docker-compose.yml

```diff
@@ -92,6 +92,7 @@ services:
     ports:
       - 4444:4444
       - 5900:5900
+      - 7900:7900
```

Comment:
Does it work to expose port 7900 on 6080 (additionally) ?

Reply:
lets give it a try 🤣

Reply:
works, done